packages/scenario/resources/ImageSelectTester.xhtml

```diff
@@ -0,0 +1,83 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml"
```

Comment:
I thought this was super weird, and I agree we should change it. If it's okay, I'd like to hold off until we're close to wrapping up review. If there are other similar changes like this, it might be nice to have a single commit of "here's some stuff we changed that isn't otherwise very visible, but JavaRosa may want to adopt the change too".